### PR TITLE
APPEALS-24145 Fix test scenario "Assigning it to complete the claims establishment"

### PR DESCRIPTION
**Pull request by seancva**
Resolves [Fix flaky/skipped tests in feature/dispatch/establish_claim_spec.rb](https://jira.devops.va.gov/browse/APPEALS-24145)

# Description
Removed a skip: "flakey hang" and added some assertions

## Acceptance Criteria
- [x] Code compiles correctly
- [x] Spec executes without failure

## Testing Plan
1. run `CI=true make one-test ./spec/feature/dispatch/establish_claim_spec.rb` in the console

### CODE REVIEW

File: spec/feature/dispatch/establish_claim_spec.rb [hunk 1]
```diff
@@ -141,6 +141,9 @@
       end
 
       visit "/dispatch/work-assignments"
+
+      expect(page).to have_current_path("/dispatch/work-assignments")
```
Comment (TylerBroyles) [inline spec/feature/dispatch/establish_claim_spec.rb:145]: All the other url expects are fine, but this one really only tests Capybara visit method.

File: spec/feature/dispatch/establish_claim_spec.rb [hunk 2]
```diff
@@ -633,6 +638,7 @@
           expect(page).to have_selector(:link_or_button, "Assign to Claim")
           click_on "Assign to Claim" # unknown reason sometimes hangs here
```
Comment (TylerBroyles) [inline spec/feature/dispatch/establish_claim_spec.rb:639]: Did we determine if it still hangs on this line or not? If it doesn't, then the comment can probably be removed.

**Review by TylerBroyles — CHANGES_REQUESTED** [review]
I have a couple of questions, but it looks good.

**Review by TylerBroyles — APPROVED** [review]
Looks good to me.

**Review by craigrva — APPROVED** [review]

**Pull request merged**